wondrous-app/components/CreateEntity/CreateEntityModal/index.tsx

```diff
@@ -0,0 +1,1224 @@
+                  }}
+                  placeholder="Enter rewards..."
+                  value={form.values?.rewards?.[0]?.rewardAmount}
+                  fullWidth={true}
+                  autoFocus={true}
+                  InputProps={{
+                    inputComponent: CreateEntityTextfieldInputRewardComponent,
+                    endAdornment: (
+                      <CreateEntityAutocompletePopperRenderInputAdornment
+                        position="end"
+                        onClick={() => {
+                          form.setFieldValue('rewards', null);
+                        }}
+                      >
+                        <CreateEntityAutocompletePopperRenderInputIcon />
+                      </CreateEntityAutocompletePopperRenderInputAdornment>
+                    ),
+                  }}
+                  error={form.errors?.rewards?.[0].rewardAmount}
+                />
+              </CreateEntityRewardWrapper>
+            )}
+            {form.touched.rewards && form.errors.rewards && (
+              <CreateEntityError>{form.errors.rewards[0].rewardAmount}</CreateEntityError>
+            )}
+            {form.values.rewards === null && (
+              <CreateEntityLabelAddButton
+                onClick={() => {
+                  form.setFieldValue('rewards', [{ rewardAmount: null, paymentMethodId: paymentMethods?.[0]?.id }]);
+                }}
+              >
+                <CreateEntityAddButtonIcon />
+                <CreateEntityAddButtonLabel>Add</CreateEntityAddButtonLabel>
+              </CreateEntityLabelAddButton>
+            )}
+          </CreateEntitySelectWrapper>
+        </CreateEntityLabelSelectWrapper>
+
+        <CreateEntityLabelSelectWrapper show={entityTypeData[entityType].fields.includes(Fields.points)}>
+          <CreateEntityLabelWrapper>
+            <CreateEntityLabel>Points</CreateEntityLabel>
+          </CreateEntityLabelWrapper>
+
+          <CreateEntitySelectWrapper>
+            {form.values.points !== null && (
+              <>
+                <CreateEntityTextfield
+                  name="points"
+                  onChange={form.handleChange('points')}
+                  fullWidth={true}
+                  value={form.values.points}
+                  InputProps={{
+                    inputComponent: CreateEntityTextfieldInputPointsComponent,
+                    endAdornment: (
+                      <CreateEntityAutocompletePopperRenderInputAdornment
+                        position="end"
+                        onClick={() => {
+                          form.setFieldValue('points', null);
+                        }}
+                      >
+                        <CreateEntityAutocompletePopperRenderInputIcon />
+                      </CreateEntityAutocompletePopperRenderInputAdornment>
+                    ),
+                  }}
+                  error={form.errors.points}
```

Comment:
In what scenario would there be an error for points? Just wondering

Reply:
- not a number
- not a whole number (if understand points, as in story points in scrum)
- negative number or zero